<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VfilterInterface</class>
 <widget class="QWidget" name="VfilterInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>741</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ScrollArea" name="VfilterscrollArea">
     <property name="minimumSize">
      <size>
       <width>760</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="VfilterfacescrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1063</width>
        <height>723</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="Vfilterbox01">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
         </property>
         <item>
          <layout class="QVBoxLayout" name="VfilterverticalLayout_4">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="VfilterTitle1">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>64</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>28</pointsize>
               <bold>true</bold>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>复杂滤镜</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="VfilterTitle2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>45</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>18</pointsize>
               <bold>true</bold>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="text">
              <string>视频</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="Vfilterlabel">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>关于复杂滤镜的处理</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="VfilterhorizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="PrimaryPushButton" name="VfilterpushBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <bold>true</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>处理视频</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="VfilterSTBtn">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>中止处理</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PushButton" name="VfilterpushBtn_2">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>解冻</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="Vfilterbox02">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>480</width>
           <height>145</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>240</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="PushButton" name="Vfilterinputfile">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>添加文件</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="PushButton" name="Vfilterinputclear">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>清除</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="Vfilteroutputfolder">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>选择输出文件夹</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="ListWidget" name="Vfilterinputlist">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>200</height>
             </size>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Vfilterbox03">
         <item>
          <widget class="QLabel" name="VcodecpIFTitle2_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>600</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>18</pointsize>
             <bold>true</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>编码设置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="VcodecpIFTitle2_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>18</pointsize>
             <bold>true</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="text">
            <string>横竖屏转换</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="Vfilterbox04">
         <item>
          <widget class="QFrame" name="Vfilterframe_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>600</width>
             <height>360</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>600</width>
             <height>360</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QGridLayout" name="VfiltergridLayout">
              <property name="horizontalSpacing">
               <number>24</number>
              </property>
              <property name="verticalSpacing">
               <number>16</number>
              </property>
              <item row="1" column="0">
               <widget class="QLabel" name="Vfilterlabel_5">
                <property name="font">
                 <font>
                  <pointsize>16</pointsize>
                  <bold>true</bold>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="text">
                 <string>自定义编码</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="2">
               <widget class="PlainTextEdit" name="VfilterplainTextEdit">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="plainText">
                 <string>-vcodec libx264 -preset medium -crf 23 -acodec aac -b:a 256k</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>(请直接从批处理面板复制自定义编码)</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="Vfilterframe_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>360</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>480</width>
             <height>360</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="1">
             <widget class="CheckBox" name="VcodecpIFcheckBox_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>音频标准化至-16LUFS</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="VfilterlineEdit_3">
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="Vfilterlabel_8">
              <property name="text">
               <string>横竖屏转换目标分辨率</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="2">
             <widget class="QPlainTextEdit" name="VfilterplainTextEdit_2">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="plainText">
               <string>滤镜参数：</string>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::TextInteractionFlag::TextEditorInteraction</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="PushButton" name="VcodecpIFClearFil">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>清除设置</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="EditableComboBox" name="VcodecpIFcutsomFilter">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>横竖屏转换</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="PushButton" name="Vfilterbgimg">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>240</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>横竖屏转换背景图</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header location="global">qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ScrollArea</class>
   <extends>QScrollArea</extends>
   <header location="global">qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>EditableComboBox</class>
   <extends>QLineEdit</extends>
   <header location="global">qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
